src/schema/borough.ts

```diff
@@ -8,6 +7,10 @@ export const borough = pgTable("borough", {
   abbr: text("abbr").notNull(),
 });
 
-export const selectBoroughSchema = createSelectSchema(borough);
+export const boroughEntitySchema = z.object({
+  id: z.string().regex(/^[0-9]$/),
+  title: z.string(),
+  abbr: z.string().length(2),
+});
 
-export type SelectBorough = z.infer<typeof selectBoroughSchema>;
+export type BoroughEntity = z.infer<typeof boroughEntitySchema>;
```

Comment:
the `[domain]EntitySchema` naming convention is specifically for zod schema representations of the table data. It is the only zod schema definition that can be stored in the `schema/[domain].ts` folder. 

The `drizzle-zod` package is removed because the schemas it creates are not precise enough.  

We need to specify that it's an 'Entity' schema to differentiate it from all the other schemas- like the repository and the query response schemas.